### PR TITLE
add tab layout in Home Screen

### PR DESCRIPTION
**Pull request by phyoe**
- add tab layout in Home screen
- create Blank Hospital Fragment 
- create Blank Clinic Fragment 
- create Blank Pharmacy Fragment

### CODE REVIEW

File: PADC-FP-HealthCareDirectory/app/src/main/java/com/padc/healthcaredirectory/activities/HomeActivity.java [hunk 1]
```diff
@@ -122,6 +128,12 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         return false;
     }
 
+    private void navigateToTabLayout() {
+        getSupportFragmentManager().beginTransaction()
+                .replace(R.id.fl_container, HealthCarePagerFragment.newInstance())
```
Comment (vincent-paing) [inline PADC-FP-HealthCareDirectory/app/src/main/java/com/padc/healthcaredirectory/activities/HomeActivity.java]: Should use Static Factory method even if there's no parameters

File: PADC-FP-HealthCareDirectory/app/src/main/java/com/padc/healthcaredirectory/fragments/HealthCarePagerFragment.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import butterknife.ButterKnife;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class HealthCarePagerFragment extends Fragment {
+
+    @BindView(R.id.tl_health_care)
+    TabLayout tlHealthCare;
```
Comment (vincent-paing) [inline PADC-FP-HealthCareDirectory/app/src/main/java/com/padc/healthcaredirectory/fragments/HealthCarePagerFragment.java:27]: I'm not sure about this naming but it looks weird. I think tablayoutHealthCare would be more self-explanatory

File: PADC-FP-HealthCareDirectory/app/src/main/res/layout/fragment_pager_health_care.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical">
+
+    <android.support.design.widget.TabLayout
+        android:id="@+id/tl_health_care"
+        app:tabIndicatorColor="@color/primary"
+        app:tabMode="fixed"
+        app:tabSelectedTextColor="@color/primary_bright"
+        app:tabTextAppearance="@android:style/TextAppearance.DeviceDefault.Widget.ActionMode.Subtitle"
```
Comment (vincent-paing) [inline PADC-FP-HealthCareDirectory/app/src/main/res/layout/fragment_pager_health_care.xml:13]: I don't think you need to change TextApperance. Is it to show MM fonts better?

File: PADC-FP-HealthCareDirectory/app/src/main/java/com/padc/healthcaredirectory/fragments/HealthCarePagerFragment.java (hunk 2 above)
Comment (phyoe) [inline PADC-FP-HealthCareDirectory/app/src/main/java/com/padc/healthcaredirectory/fragments/HealthCarePagerFragment.java]: Yes, I will change the name of variable.

File: PADC-FP-HealthCareDirectory/app/src/main/res/layout/fragment_pager_health_care.xml (hunk 3 above)
Comment (phyoe) [inline PADC-FP-HealthCareDirectory/app/src/main/res/layout/fragment_pager_health_care.xml]: I am not sure. I will check this out, Bro. Thz.

**Pull request merged**